src/main/java/org/apache/hadoop/io/MaxTextLineLengthExceededIOException.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.hadoop.io;
```

Comment:
Since this is our custom exception, we should put it in our own package `com.facebook.presto.hadoop` in order to avoid conflicts with Hadoop.